documentation changes (#698)
anoma/namada: documentation/docs/src/quick-start.md

# can detach the tmux (Ctrl-B then D)
```

For a more verbose output, one can run 
```shell
export ANOMA_TM_STDOUT='true'
namada ledger
```

This should sync your node to the ledger and will take a while (depending on your computer). Subsequent commands (generating an account, etc.)  are unlikely to work until it is fully synced. Enquire the current block height with other participants to make sure you are synced in order to proceed.

## Account

Accounts on Namada are divided into two subcategories:

> * Implicit accounts (no attached Validity Predicate)
> * Established accounts (attached Validity Predicate)

Reviewer: ```suggestion
* Implicit accounts (all use the same Validity Predicate)
* Established accounts (can use a chosen Validity Predicate)
```

> In order to make transactions on Namada, an implicit account is required.

Reviewer 2: ```suggestion
In order to make transactions on Namada, an account is required, and an implicit account can be created easily.
```
someone can make transactions using an established account as well (afaik?)

An implicit account can be generated from the commands described below

```shell
namadaw address gen \
  --alias example-implicit

➜ Enter encryption password: 
Successfully added a key and an address with alias: "example-implicit"

```
An implicit account with alias "example-implicit" has now been successfully generated and stored locally in the wallet.

> It is possible to "upgrade" this implicit account to become an Established Account through submitting a transaction to do so. Note that this transaction, as with any transaction on Namada, will consume "gas", and will require a positive balance of NAM tokens.

Reviewer 2: idk if "upgrade" would be the best term here, you can create an established account using an implicit account, but the implicit account still remains in existence in your wallet, there is no replacement of it.

Author: Very good point actually

> It is possible to "upgrade" this implicit account to become an Established Account through submitting a transaction to do so. Note that this transaction, as with any transaction on Namada, will consume "gas", and will require a positive balance of NAM tokens.

Reviewer 2: ```suggestion
It is possible to "upgrade" this implicit account to become an Established Account through submitting a transaction to do so. Note that this transaction, as with any transaction on Namada, will consume "gas", and will require a positive balance of NAM tokens for the implicit account which is being used to create the established account.
```

Reviewer 2: btw do you know if implicit accounts are charged gas now? i.e. when an implicit account makes a transaction, is NAM actually deducted from its balance?
